### PR TITLE
chore: update cmdlet names

### LINKED ISSUE #73: Ensure consistent naming across cmdlets
### Code of Conduct

- [X] I have read and agree to the project's Code of Conduct.
- [X] Vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue initial description to help the maintainers prioritize.
- [X] Do not leave "+1" or other comments that do not add relevant information or questions.
- [X] If you are interested in working on this issue or have submitted a pull request, please leave a comment.

### Description

With the change from #68, there was a little inconsistency in the name of the cmdlets. As a result, it was discussed to have a consistent naming across the cmdlets. 

Hence -

Rename `Set-SddcCertiticateAuthority` to` Set-VCFCertiticateAuthority`.
Rename` Request-SddcCertificate` to `Request-VCF***`

### Use Case(s)

NA

### Potential Configuration

NA

### References

NA

### PR DESCRIPTION
**Pull request by bhumitra**
### Summary

- **Breaking Change**: Renamed `Set-SddcCertificateAuthority` to `Set-VCFCertificateAuthority`
- **Breaking Change**: Renamed `Request-SddcCertificate` to `Request-VCFSignedCertificate` 
- Updated docs as per above changes. 

### Type

- [ ] Bugfix
- [x] Enhancement or Feature
- [ ] Code Style or Formatting
- [ ] Documentation
- [ ] Refactoring
- [ ] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

- [x] Yes, there are breaking changes.
- [ ] No, there are no breaking changes.

### Test and Documentation

- [x] Tests have been completed.
- [x] Documentation has been added or updated.

### Issue References

Closes #73

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,8 @@
 > Release Date: Unreleased
 
 Enhancement:
-
+- **Breaking Change**: Renamed `Set-SddcCertificateAuthority` to `Set-VCFCertificateAuthority`
```
Comment (tenthirtyam) [inline CHANGELOG.md:8]:
```suggestion
- **Breaking Change**: Renamed `Set-SddcCertificateAuthority` to `Set-VCFCertificateAuthority`. [GH-74](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-certificate-management/pull/74)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,8 @@
...
+- **Breaking Change**: Renamed `Request-SddcCertificate` to `Request-VCFSignedCertificate`
```
Comment (tenthirtyam) [inline CHANGELOG.md:9]:
```suggestion
- **Breaking Change**: Renamed `Request-SddcCertificate` to `Request-VCFSignedCertificate`. [GH-74](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-certificate-management/pull/74)
```

File: VMware.CloudFoundation.CertificateManagement.psm1 [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 VMware, Inc.
+ # Copyright 2023 VMware, Inc.
```
Comment (tenthirtyam) [inline VMware.CloudFoundation.CertificateManagement.psm1:1]:
```suggestion
# Copyright 2023 VMware, Inc.
```

File: VMware.CloudFoundation.CertificateManagement.psm1 [hunk 4]
```diff
@@ -1819,13 +1823,18 @@ Function Request-SddcCertificate {
 
         .PARAMETER workloadDomain
         The name of the workload domain in which the certificate is requested to be signed.
+
+        .PARAMETER certAuthority
+        The type of Certificate Authority to be used for signing certificates - Microsoft or OpenSSL
```
Comment (tenthirtyam) [inline VMware.CloudFoundation.CertificateManagement.psm1:1828]:
```suggestion
        The type of Certificate Authority to be used for signing certificates. One of: Microsoft, OpenSSL.
```

**Review by tenthirtyam — CHANGES_REQUESTED** [review]
Minor changes.

**Comment by bhumitra** [comment]
Applied suggested changes.

**Review by tenthirtyam — APPROVED** [review]
LGTM

**Pull request merged**